javascript/package.json

```diff
@@ -0,0 +1,45 @@
+{
+    "name": "creditcoin-ts",
+    "version": "0.0.1",
+    "description": "Polkadot JS examples and integration tests for Creditcoin Substrate",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/gluwa/creditcoin-substrate.git"
+    },
+    "license": "Unlicense",
+    "bugs": {
+        "url": "https://github.com/gluwa/creditcoin-substrate/issues"
+    },
+    "homepage": "https://github.com/gluwa/creditcoin-substrate#readme",
+    "packageManager": "yarn@3.2.0",
+    "scripts": {
+        "start": "npx ts-node -O '{\"module\":\"commonjs\"}' ./src/index.ts",
+        "lint": "tsc --noEmit --pretty",
+        "test": "polkadot-dev-run-test --forceExit --runInBand",
+        "build:types": "yarn generate:defs && yarn generate:meta",
```

Comment:
does this work? it is using ts-node without npx.

Reply:
You mean line 19? It does work, yes. It's executed from `prepare-devel-env.sh`.

Reply:
Hmmm. I must have done something wrong.  Tried removing `npx` from line 16 and `yarn start` again and it works. Will push an update.